internal/cli/install/install.go

```diff
@@ -56,30 +55,59 @@ func (o *InstallOptions) Parse(_ *cobra.Command, args []string) error {
 }
 
 // Install component
-func (o *InstallOptions) Install() error {
+func Install(component string, version string) error {
 	var url string
-	baseUrl := "https://raw.githubusercontent.com/oceanbase/ob-operator/"
-	for component, version := range o.Components {
-		switch component {
-		case "cert-manager":
-			componentFile := "cert-manager.yaml"
-			url = fmt.Sprintf("%s%s/deploy/%s", baseUrl, version, componentFile)
-		case "ob-operator":
-			componentFile := "operator.yaml"
-			url = fmt.Sprintf("%s%s/deploy/%s", baseUrl, version, componentFile)
-		}
-		if err := run(url); err != nil {
+	var cmd *exec.Cmd
+	obUrl := "https://raw.githubusercontent.com/oceanbase/ob-operator/"
+	localPathUrl := "https://raw.githubusercontent.com/rancher/local-path-provisioner/"
+	switch component {
+	case "cert-manager":
+		componentFile := "cert-manager.yaml"
+		url = fmt.Sprintf("%s%s/deploy/%s", obUrl, version, componentFile)
+		cmd = exec.Command("kubectl", "apply", "-f", url)
+	case "ob-operator", "ob-operator-dev":
+		componentFile := "operator.yaml"
+		url = fmt.Sprintf("%s%s/deploy/%s", obUrl, version, componentFile)
+		cmd = exec.Command("kubectl", "apply", "-f", url)
+	case "local-path-provisioner", "local-path-provisioner-dev":
+		componentFile := "local-path-storage.yaml"
+		url = fmt.Sprintf("%s%s/deploy/%s", localPathUrl, version, componentFile)
+		cmd = exec.Command("kubectl", "apply", "-f", url)
+	case "ob-dashboard":
+		if err := preRunCmd(); err != nil {
 			return err
 		}
+		versionFlag := fmt.Sprintf("--version=%s", version)
+		cmd = exec.Command("helm", "install", "oceanbase-dashboard", "ob-operator/oceanbase-dashboard", versionFlag)
+	}
+	return runCmd(cmd)
+}
+
+// preRunCmd preRun two commands for ob-dashboard installation
+func preRunCmd() error {
```

Comment:
Maybe this function can have a better name, this function is just called when installing dashboard, maybe 'updateHelmRepo' is a better choice

Reply:
Ok, it has been changed.